dotnet/targets/Xamarin.Shared.Sdk.targets

```
@@ -1189,6 +1193,8 @@
 			<_MainLinkerFlags Include="-liconv" />
 			<_MainLinkerFlags Include="-lcompression" />
 
+			<_MainLinkerFlags Condition="'$(_ExportSymbolsExplicitly)' == 'true'" Include="-exported_symbols_list $(_ExportedSymbolsFile)" />
```

Each argument is a separate item in the item group, so you need to add twice:

```suggestion
			<_MainLinkerFlags Condition="'$(_ExportSymbolsExplicitly)' == 'true'" Include="-exported_symbols_list" />
			<_MainLinkerFlags Condition="'$(_ExportSymbolsExplicitly)' == 'true'" Include="$(_ExportedSymbolsFile)" />
```

This is necessary to support paths with spaces.

---

Thanks. Fixed.